tests/core/full_node/full_sync/test_full_sync.py

```diff
@@ -208,6 +209,46 @@ def fn3_is_not_syncing():
         await time_out_assert(180, node_height_exactly, True, full_node_1, 999)
         await time_out_assert(180, node_height_exactly, True, full_node_2, 999)
 
+    @pytest.mark.asyncio
+    async def test_sync_none_wp_response_backward_comp(self, three_nodes, default_1000_blocks):
+        num_blocks_initial = len(default_1000_blocks) - 50
+        blocks_950 = default_1000_blocks[:num_blocks_initial]
+        full_node_1, full_node_2, full_node_3 = three_nodes
+        server_1 = full_node_1.full_node.server
+        server_2 = full_node_2.full_node.server
+        server_3 = full_node_3.full_node.server
+        server_3.set_capabilities([(uint16(Capability.BASE.value), "1")])
+
+        for block in blocks_950:
+            await full_node_1.full_node.respond_block(full_node_protocol.RespondBlock(block))
+
+        await server_2.start_client(PeerInfo(self_hostname, uint16(server_1._port)), full_node_2.full_node.on_connect)
+        await server_3.start_client(PeerInfo(self_hostname, uint16(server_1._port)), full_node_3.full_node.on_connect)
+
+        # Also test request proof of weight
+        # Have the request header hash
+        peers: List = [c for c in full_node_2.full_node.server.all_connections.values()]
+        request = full_node_protocol.RequestProofOfWeight(
+            blocks_950[-1].height + 1, default_1000_blocks[-1].header_hash
+        )
+        start = time.time()
+        res = await peers[0].request_proof_of_weight(request, timeout=5)
+        assert res.data == b""
```

Comment:
Would be good to separate testing the `b""` case and an invalid capability case, wherethe capability is outside the valid range, > max_valid_capability.

Reply:
checking b"" is a way for me to differentiate the flow between the old capabilities and the new, so we check that, about the out of range i think this is already tested, the capabilities code is not new but il try and find if we do that